src/app/postulants/postulants.component.html

```diff
@@ -60,17 +60,14 @@ <h3>Postulants</h3>
                 postulant.phone
               }}</a>
             </td>
-            <td class="center-align">
-              <label>
-                <input
-                  type="checkbox"
-                  [checked]="postulant.validated"
-                  (change)="
-                    postulantsService.acceptPostulant(currentUser, postulant)
-                  "
-                />
-                <span></span>
-              </label>
+            <td class="hide-on-small-only">
+              <a
+                href="{{ postulant.transferSupportURL }}"
```

Comment:
Only a suggestion, prefer property binding notation over string interpolation when it's easier to use that: `[href]="postulant.transferSupportURL"`